cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/co/cask/cdap/etl/spark/streaming/DefaultStreamingContext.java

```diff
@@ -94,4 +97,9 @@ public void execute(TxRunnable runnable) throws TransactionFailureException {
   public void execute(int timeout, TxRunnable runnable) throws TransactionFailureException {
     sec.execute(timeout, runnable);
   }
+
+  @Override
+  public void record(List<Operation> operations) {
+    throw new UnsupportedOperationException("Lineage recording is not supported.");
```

Comment:
We discussed earlier about supporting field lineage for the streaming pipelines with respect to following issues -
1. lifecycle for the streaming pipelines is different than batch, for example we record the lineage once the batch program is complete, however there is no completion of the streaming pipelines
2. because of checkpointing, once started, runtime arguments of the streaming pipelines cannot be changed, so each run will record the same set of operations

We decided to do the work for batch first. I have filed JIRA to track this issue - https://issues.cask.co/browse/CDAP-13643

Reply:
ok